todo_app/app.py

```diff
@@ -1,25 +1,60 @@
-
+import json
```

Comment:
You've gone back to modifying the `app.py` file, which means that this doesn't work with the Mongo changes you made recently! We should bring those two together; probably by moving the Mongo functionality into this file

Reply:
Deleted mongo file as this is using Trello for this exercise and uses Mongo for Module-11